apps/src/AzureTextToSpeech.js

```diff
@@ -74,9 +73,11 @@ export default class AzureTextToSpeech {
    * @param {string} opts.ssml SSML in request body.
    * @param {string} opts.token Authentication token from Azure.
    * @param {function(Array<string>)} opts.onProfanityFound Called if the given text contains profanity.
-   * @returns {Promise<SoundResponse>} A promise that returns a SoundResponse when resolved.
+   * @returns {function} A thunk that returns promise, which resolves to a SoundResponse. Example usage:
+   * const soundPromise = createSoundPromise(options);
+   * const soundResponse = await soundPromise();
    */
-  createSoundPromise = opts => {
+  createSoundPromise = opts => () => {
```

Comment:
this is the actual change 😄  everything else in this PR is updating consumers, documentation, and unit tests

Reply:
thank you for this comment!! made this much easier to review :)